README.md

```diff
@@ -14,7 +14,7 @@ I started this project to learn about reverse engineering big Javascript codebas
 
 If you benefit from this project, you can buy me a beer :beers: :+1:
 
-[![paypal](https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif)](https://www.paypal.me/timklingeleers)
+[![paypal](https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif)](https://www.paypal.me/eimarfandino)
```

Comment:
Ok!

Reply:
I only changes this since we discussed about the maintainer role, but i agree with you. Lets first put it through so people can start having fun and we then discuss this another time